src/libsystemd/sd-netlink/netlink-message.c

```diff
@@ -830,7 +830,7 @@ int sd_netlink_message_read_in6_addr(sd_netlink_message *m, unsigned short type,
 static int netlink_container_parse(sd_netlink_message *m,
                                    struct netlink_container *container,
                                    struct rtattr *rta,
-                                   unsigned rt_len) {
+                                   unsigned short rt_len) {
```

Comment:
actually I think we should always use size_t when talking about sizes in memory. If some kernel headers are confused and use something else that's fine, and we should cast it to size_t before using it if necessary. 

Hence, I think this should be changed to size_t here, and instead we should explicitly cast to size_t where the checker complains

Reply:
But the checker is in the macro `RTA_OK()`... So, let's ignore the warning ;-)